lib/collections/addon/components/collections-submission/collection-submission-confirmation-modal/component.ts

```diff
@@ -1,9 +1,12 @@
 import Component from '@ember/component';
+import { tracked } from '@glimmer/tracking';
 
 import { layout } from 'ember-osf-web/decorators/component';
 
+import style from './styles';
 import template from './template';
 
-@layout(template)
+@layout(template, style)
```

Comment:
I wonder if it would be better just to make this a Glimmer component rather than a classic component. Then you wouldn't need to do this at all, plus it brings it into a more modern standard.